package/CHANGELOG

```diff
@@ -78,6 +78,7 @@ Fixes
   * Added missing selection module to leaflet.py (Issue #2612)
 
 Enhancements
+  * Added .frames and .times arrays to AnalysisBase (Issue #2661)
```

Comment:
Thoughts on also introducing a CHANGELOG entry under `changes` for hbond_analysis?

Reply:
I would think a CHANGELOG entry would be about the same thing as a `.. versionchanged::` note wrt updating a new class, though.